src/xtgeo/grid3d/_gridprops_import_eclrun.py

```diff
@@ -62,7 +62,6 @@ def sanitize_date_list(
 
 
 def import_ecl_init_gridproperties(
```

Comment:
The docstring is a bit outdated.

Would be nice if the return was typed as `List[GridProperty]`.

Reply:
Done

Reply:
Docstring describes the parameter `dates` which is no longer needed.